### PR TITLE
Task7

### PR DESCRIPTION
**Pull request by Yura935**
База даних уже була імплементована в pull request #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Project: Messenger with translating(Bachelor work)
...
  - Firebase — це набір інструментів для «створювання, покращення та розвитку вашого додатка», а інструменти, які він надає, охоплюють значну частину сервісів, які розробникам зазвичай доводиться створювати самостійно, але насправді вони не хочуть створювати, оскільки вони краще зосередитися на самому додатку. Це включає такі речі, як аналітика, аутентифікація, бази даних, конфігурація, зберігання файлів, push-повідомлення, і список можна продовжувати. Служби розміщуються в хмарі та масштабуються без будь-яких зусиль з боку розробника.
  - Для підключення Firebase до нашого проекту треба в файл app.module.ts заімпортувати відповідні модулі(AngularFireModule - для загального підключення БД, AngularFirestoreModule - для  підключення зберігання даних в колекціях, AngularFireStorageModule - для зберігання фото і інших файлів у бд, AngularFireAuthModule - для роботи з логінуванням і реєстрацією).
- - Для даного проекту було створено 1 колекцію (users) - це набір користувачів нашогої системи. Кожен користувач є записаний в колекцію за своїм id. В кожного користувача є такі поля: email, image, userName, contacts, id. Розберемо кожен детально.
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md]: What is this? what should I review?

Comment (Yura935) [reply README.md]: As I wrote in the readme file, I used a non-relational database - Firebase. It has no links and tables as such, and data is stored in collections - arrays of objects. And those in turn can contain any type of data(string, number, object, array, date ets.). And values from the database are returned in JSON format.

File: README.md [hunk 2]
```diff
@@ -42,6 +42,18 @@ Project: Messenger with translating(Bachelor work)
  - Також було дописано Unit-тести для нового функціоналу. Зараз проект покрито десь на 65-70%, що досить добре.
  - Також було відформатовано і відредаговано деякі частини коду.
 
+ # Messanger v2.0 БД
```
Comment (andrii-s-shpitser-lpnu-ua) [inline README.md:45]: where is task#6?

Comment (Yura935) [reply README.md:45]: I haven't done task 6 yet. I asked you if I could pass task 7 first. And you let me.

**Review by andrii-s-shpitser-lpnu-ua — APPROVED** [review]

**Pull request merged**